tests/pipelines/test_vqrf.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) Alibaba, Inc. and its affiliates.
+import os
+import unittest
+
+import torch
+
+from modelscope.hub.snapshot_download import snapshot_download
+from modelscope.msdatasets import MsDataset
+from modelscope.outputs import OutputKeys
+from modelscope.pipelines import pipeline
+from modelscope.utils.constant import DownloadMode, Tasks
+from modelscope.utils.test_utils import test_level
+
+
+class NeRFReconVQCompressionBlender(unittest.TestCase):
+
+    def setUp(self) -> None:
+        self.model_id = 'DAMOXR/cv_nerf_3d-reconstruction_vector-quantize-compression'
+        pretrained_model = 'ficus_demo.pt'
+        data_dir = MsDataset.load(
+            'nerf_recon_dataset', namespace='damo',
+            split='train').config_kwargs['split_config']['train']
+        nerf_synthetic_dataset = os.path.join(data_dir, 'nerf_synthetic')
+        self.blender_scene = 'ficus'
+        data_dir = os.path.join(nerf_synthetic_dataset, self.blender_scene)
+
+        self.pipeline = pipeline(
+            Tasks.nerf_recon_vq_compression,
+            model=self.model_id,
+            dataset_name='blender',
+            data_dir=data_dir,
+            downsample=1,
+            ndc_ray=False,
+            ckpt_path=pretrained_model)
+
+    @unittest.skipUnless(test_level() >= 2, 'skip test in current test level')
```

Comment:
setting test_level() >=2 will skip this test ,  if you want to run in in ci test, settting test_level() >=0